Kickstarter-iOS/Views/Storyboards/RewardPledge.storyboard

```diff
@@ -656,7 +650,6 @@
     </scenes>
     <resources>
         <image name="checkmark-icon" width="11" height="9"/>
-        <image name="close-icon" width="12" height="12"/>
```

Comment:
The storyboard had a custom close button so removing it brings back the native back button.